tensorboardX/proto/api.proto

```diff
@@ -104,6 +104,7 @@ enum DataType {
   DATA_TYPE_STRING = 1;
   DATA_TYPE_BOOL = 2;
   DATA_TYPE_FLOAT64 = 3;
+  DATA_TYPE_CALLABLE = 4;
```

Comment:
I think we should stay on DATA_TYPE_STRING, this additional DataType might conflict with the official TensorBoard if they adds another type in the future.